### PR TITLE
Created Concept Entry for Python SQL Connectors

### LINKED ISSUE #5633: [Concept Entry] Python SQL Connectors
### Reason/inspiration (optional)

We would like a new entry on the `SQL Connectors` concept under Python. The entry should go in a new file under `docs/content/python/concepts/sql-connectors/sql-connectors.md`.

The entry should include:  
  
- An introduction to the concept  
- A `Syntax` section that provides the syntax for the concept  
- An `Example` section that provides an example demonstrating the concept in use  
- A `Codebyte Example` section that provides a codebyte example demonstrating the concept in use  
  
Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entry for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by Rendon-04**
### Issue Solved

Closes #5633

### Type of Change

This pull request adds documentation for the `mysql-connector-python` library, including examples for setting up connections and executing SQL queries.

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/python/concepts/sql-connectors/sql-connectors.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  - 'Python'
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+**SQL Connectors** are libraries that enable Python applications to connect to and interact with SQL databases. They serve as a bridge, allowing Python code to execute SQL commands for tasks like performing insert, delete, update, and read operations on a SQL Server database. By using SQL connectors, developers can dynamically interact with databases while maintaining the original database structure.
+
+## Setting Up a SQL Connector 
+To use a SQL connector, you first need to set up your development environment. This includes installing the appropriate connector library for you database and making sure that youre database server is accessible.
```
Comment (SaviDahegaonkar) [inline content/python/concepts/sql-connectors/sql-connectors.md:20]:
```suggestion
## Setting Up a SQL Connector 
To use a SQL connector, you first need to set up your development environment. This includes installing the appropriate connector library for your database and ensuring that your database server is accessible. 
```
Made the sentence more readable by correcting the grammatical mistakes.

File: content/python/concepts/sql-connectors/sql-connectors.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+- For MySQL: Use ```pip install mysql-connector-python```.
+- For PostgreSQL: Use ```pip install psycopg2```
+- For SQLite: No installation is needed; it is built into Python's standard library.
+
+2. Set Up the Database
+- Make sure the database server is running and accessible. 
+- Create or access an exisiting database. 
+
+3. Import the Connector Library
+Use the appropriate Python module to work with the database, such as ```mysql.connector``` for MySQL or ```sqlite3``` for SQLite.
```
Comment (SaviDahegaonkar) [inline content/python/concepts/sql-connectors/sql-connectors.md:31]:
```suggestion
1. Install the Connector:
- For MySQL: Use ```pip install mysql-connector-python```.
- For PostgreSQL: Use ```pip install psycopg2```.
- For SQLite: No installation is needed; it is built into Python's standard library.

2. Set Up the Database
- Ensure the database server is running and accessible. 
- Create or access an existing database. 

3. Import the Connector Library
Use the appropriate Python module to work with the database, such as ```mysql.connector``` for MySQL or ```sqlite3``` for SQLite. 
```

File: content/python/concepts/sql-connectors/sql-connectors.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+**General Workflow:**
+- Import the connector library. 
+- Use the ```connect()``` method to establish a connection.
+- Handle connection errors with try-except blocks. 
+
+**General Workflow:**
+- Import the connector library.
+- Use the `connect()` method to establish a connection.
+- Handle connection errors with try-except blocks.
```
Comment (SaviDahegaonkar) [inline content/python/concepts/sql-connectors/sql-connectors.md:44]:
```suggestion
```
Removed the redundant  part

File: content/python/concepts/sql-connectors/sql-connectors.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+```
+
+## Executing SQL Queries
+
+Once connected, use a cursor object to execute SQL queries. The cursor facilitates operations such as creating tables, inserting data, and retrieving records.
+
+**Query Execution:**
+
+- Use `cursor.execute()` to run queries.
+- Use `cursor.fetchall()` to fetch data for SELECT queries.
```
Comment (SaviDahegaonkar) [inline content/python/concepts/sql-connectors/sql-connectors.md:66]:
```suggestion
**Query Execution:**

- Use `cursor.execute()` to execute SQL queries.
- Use `cursor.fetchall()` to fetch all rows from a `SELECT` query.
```

**Review by SaviDahegaonkar — APPROVED** [review]

**Review by PragatiVerma18 — APPROVED** [review]
This looks good to go! 🚀

**Pull request merged**